RevenueCatUI/CustomerCenter/Views/RestorePurchasesAlert.swift

```diff
@@ -83,7 +83,7 @@ struct RestorePurchasesAlert: ViewModifier {
 
                 case .purchasesNotFound:
                     let message = Text(localization.commonLocalizedString(for: .purchasesNotRecovered))
-                    if let url = supportURL {
+                    if let url = supportURL, !customerCenterViewModel.appUpdateRequiredToContactSupport {
```

Comment:
Hmm do we need to show the `AppUpdateWarningView` in the else case? As in, right now we won't be able to contact support if the app update is required right? Which I think we didn't want to do? Same for the `WrongPlatformView`.